.ci/README.md

```diff
@@ -11,7 +11,7 @@ Mellanox Open MPI CI includes:
 * Sanity functional testing.
 ### How to Run CI
 Mellanox Open MPI CI is triggered upon the following events:
-* Push a commit into the master branch. CI status and log files are available on the Azure DevOps server.
+* Push a commit into the master branch or release branches (starting from v4.0.x). CI is started automatically. CI status and log files are available on the Azure DevOps server.
```

Comment:
FWIW, it looks like your `azure-pipelines.yml` file will take `v*.*.x`, which isn't just restricted to >= v4.0.0.

Reply:
Oh, never mind.  I see that this commit is on master and #7351 is on the v4.0.x branch.  Got it.

Reply:
CI will be run only if there's `azure-pipelines.yml` on that branch. For example, CI won't be run on `v3.1.x` branch (neither per-commit nor for PRs for `v3.1.x`) just because there isn't `azure-pipelines.yml` file in `v3.1.x`.